src/cascadia/TerminalApp/ActionArgs.h

```diff
@@ -241,6 +242,53 @@ namespace winrt::TerminalApp::implementation
             return *args;
         }
     };
+
+    // Possible SplitState values
+    // TODO:GH#2550/#3475 - move these to a centralized deserializing place
+    static constexpr std::string_view VerticalKey{ "vertical" };
+    static constexpr std::string_view HorizontalKey{ "horizontal" };
+    static TerminalApp::SplitState ParseSplitState(const std::string& stateString)
+    {
+        if (stateString == VerticalKey)
+        {
+            return TerminalApp::SplitState::Vertical;
+        }
+        else if (stateString == HorizontalKey)
+        {
+            return TerminalApp::SplitState::Horizontal;
+        }
+        // default behavior for invalid data
+        return TerminalApp::SplitState::None;
+    };
+
+    struct SplitPaneArgs : public SplitPaneArgsT<SplitPaneArgs>
+    {
+        SplitPaneArgs() = default;
+        GETSET_PROPERTY(winrt::TerminalApp::SplitState, SplitStyle, winrt::TerminalApp::SplitState::None);
+
+        static constexpr std::string_view StyleKey{ "split" };

```

Comment:
```suggestion
        static constexpr std::string_view SplitKey{ "split" };
```
I hate to be a pain, but maybe we should change the vars from "*Style*" to "*Split*"